developer-manual/development_process.rst

```diff
@@ -128,10 +151,12 @@ The *Developer*.
 
 * Finally, clears the *Assignee*.
 
-If the issue is not valid, it can be closed adding one of the labels **invalid**,
-**duplicate**, **wontfix**.
+If the issue is not valid, it can be closed using the **invalid** label
+and adding the reason inside the description like *duplicate* or *wontfix*.
```

Comment:
```suggestion
A comment must convey the reason why it is invalid, like *duplicate of (URL of issue)*, *wontfix because ...".
```

Reply:
Because I want to know who is working on a draft, otherwise I will fell free to close it :evi